examples/dodge-the-creeps/rust/src/hud.rs

```diff
@@ -26,12 +26,17 @@ impl Hud {
     pub fn show_game_over(&self) {
         self.show_message("Game Over".into());
 
+        let mut timer = self.base.get_tree().unwrap().create_timer(2.0).unwrap();
+        timer.connect("timeout".into(), self.base.callable("_show_start_button"));
+    }
+
+    #[func]
+    fn _show_start_button(&mut self) {
```

Comment:
In Rust, underscores at the beginning of symbols means "unused", while in GDScript it means "private" (by convention). Since we have better means to convey visibility, we don't generally use this convention.

Reply:
That's strange, because the `#[func]` macro should expsnd to code that _does_ use the function. We have dozens of tests doing this, and even clippy is fine with it.

What warnings do you get?

Reply:
Looks like my editor was drunk. I'll go with non `_` naming. I'll clean this up per contrib guidelines and reopen.